tests/ui/conftest.py

```diff
@@ -19,13 +14,14 @@ def firefox_options(firefox_options):
 
     """
     firefox_options.set_preference(
-        'extensions.install.requireBuiltInCerts', False)
```

Comment:
All these options still need some documentation, as discussed, can you add that please?

Reply:
Can you either open an issue about that or simply add comments that document these settings one by one?